gradle/publish.gradle

```diff
@@ -13,14 +14,15 @@ task testJar(type: Jar) {
 
 publishing {
     publications {
-        bintray(MavenPublication) {
+        artifactory(MavenPublication) {
             from components.java
             artifact(sourceJar) {
                 classifier 'sources'
             }
             artifact(testJar) {
                 classifier 'tests'
             }
+            version project.version.replaceAll(/-\d+.*/, '-SNAPSHOT')
```

Comment:
This is a simple way to determine that we are not publishing a release artifact which will be of the x.x.x(-alpha|-beta)+(some digits)